### PR TITLE
[Needs Review] Issue command bug fixes

### PR DESCRIPTION
**Pull request by rudehn**
The ability to send commands to chaosbot via issue comments is now available. However there were a few major bugs with the first version.

This PR..

- [x] Runs commands from oldest to newest
- [x] Has slightly more error handling
- [x] Better validation of commands. `/vote for :beers:` will no longer be run.
- [x] Stop overwriting command comment. Bot would overwrite comment with command saying "Command has been run". This now updates on the bot status reply instead.
- [x]  Don't show negative time remaining

### CODE REVIEW

**Comment by PlasmaPower** [comment]
I'll approve after lints are fixed.

**Comment by phil-r** [comment]
> /vote for :beers: will no longer be run.

😭

File: cron/poll_read_issue_comments.py [hunk 1]
```diff
@@ -246,26 +237,51 @@ def handle_comment(api, issue_comment):
...
-            db = update_db(global_comment_id, {"has_ran": True})
-            db_fields = select_db(global_comment_id, ("chaos_response_id", "command"), db=db)
-            resp_id = db_fields["chaos_response_id"]
-            command = db_fields["command"]
-            body = "> {command}\n\nVote failed".format(command=command)
-            gh.comments.edit_comment(api, settings.URN, resp_id, body)
+            update_command_ran(api, global_comment_id, "Vote Failed")
 
 
 def is_command(comment):
```
Comment (phil-r) [inline cron/poll_read_issue_comments.py:253]: That would be cool to have some tests for this ;)

File: cron/poll_read_issue_comments.py [hunk 2]
```diff
@@ -138,7 +130,7 @@ def has_enough_votes(votes):
     # At least one negative vote will cause vote to not pass
     for user, vote in votes.items():
         if vote < 0:
-            __log.debug("vote less than one")
+            # __log.debug("vote less than one")
```
Comment (mark-i-m) [inline cron/poll_read_issue_comments.py:133]: No strong feelings, but could you remove these?

Comment (rudehn) [reply cron/poll_read_issue_comments.py:133]: yeah those were for debugging. I can pop them out in the next version

**Comment by mark-i-m** [comment]
Nice job at @rudehn :beers: 
Much needed improvements!

File: cron/poll_read_issue_comments.py (hunk 1 above)
Comment (rudehn) [reply cron/poll_read_issue_comments.py:253]: I'll look into getting some. The next step is to refactor the code to use mysql. Should make everything cleaner and easier to debug

**Comment by mark-i-m** [comment]
@PlasmaPower Meritocracy?

**Review by PlasmaPower — APPROVED** [review]

**Comment by mark-i-m** [comment]
@rudehn The title of this PR is inaccurate. This is unacceptable. Remove [Needs Review] this instant!!! :rage:

**Review by andrewda — APPROVED** [review]

**Comment by phil-r** [comment]
🎉 That did work!

**Pull request merged**